### PR TITLE
Team1 Spinnaker helm chart  

### PR DESCRIPTION
**Pull request by aizada92**
Hey  ! Please take a look our Spinnaker, and please accept this request !

### CODE REVIEW

**Comment by OlgaLuchian** [comment]
Great job !!!!

File: charts/spinnaker/charts/minio/values.yaml [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+  type: ClusterIP
+  clusterIP: None
+  port: 9000
+  # nodePort: 31311
+  annotations: {}
+    # prometheus.io/scrape: 'true'
+    # prometheus.io/path:   '/minio/prometheus/metrics'
+    # prometheus.io/port:   '9000'
+
+ingress:
```
Comment (Murodbey) [inline charts/spinnaker/charts/minio/values.yaml:80]: This part needs to be revieview

File: charts/spinnaker/values.yaml [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+  # A list of resource kinds this Spinnaker account cannot deploy to or cache.
+  # This can only be set when –kinds is empty or not set.
+  # omittedKinds:
+  # -
+
+# Change this if youd like to expose Spinnaker outside the cluster
+ingress:
+  enabled: true
+  host: build.fuchicorp.com
```
Comment (Murodbey) [inline charts/spinnaker/values.yaml:202]: spinnaker.fuchicorp.com

File: charts/spinnaker/values_saml.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# Configure ingress to allow access to both gate and deck from your machine:
+ingress:
```
Comment (Murodbey) [inline charts/spinnaker/values_saml.yaml:2]: do we need 2 different values.yaml  files?

File: charts/spinnaker/values.yaml (hunk 2 above)
Comment (Murodbey) [reply charts/spinnaker/values.yaml:202]: ${deployment_endpoint} 
using variable is even better

**Pull request merged**